app/screens/settings/Settings.tsx

```diff
@@ -636,10 +636,11 @@ class Settings extends Component<Props, State> {
   }
 
   setSkin = ({ index }: { index: number }) => {
-    const { switchSkin } = this.props;
     setClientSettingsTheme(index.toString());
+    // Please refrain from uncommenting 'switchSkin' to avoid potential issues with reusing the function within the same file, as it could disrupt the 'this' reference.
     // @ts-ignore
-    switchSkin(index.toString());
+    // eslint-disable-next-line react/destructuring-assignment
+    this.props.switchSkin(index.toString());
```

Comment:
It does not solve the problem

<img width="1442" alt="image" src="https://github.com/spacemeshos/smapp/assets/1897530/e36549ab-72e3-4d43-8503-7b744ae7f16c">

Reply:
To avoid blocking PR with this thing — I'd propose just omitting this commit and then figuring out what is wrong here in a separate PR